dist/webinstaller/make-webinstaller-manifest.js

```diff
@@ -0,0 +1,63 @@
+// Copyright (C) 2020  Matthew "strager" Glazar
+// See end of file for extended copyright information.
+
+const fs = require("fs");
+
+const args = process.argv.slice(2);
+const releaseVersion = process.argv[3];
+const releaseDate = process.argv[5];
+const outputFile = process.argv[7];
+
+if (!args.includes("-releaseVersion")) {
+  console.log("error: missing -releaseVersion");
+  return 0;
+}
+if (!args.includes("-releaseDate")) {
+  console.log("error: missing -releaseDate");
+  return 0;
+}
+if (!args.includes("-Out")) {
+  console.log("error: missing -Out");
+  return 0;
+}
+
+const releaseVersionRegex = /^\d+\.\d+\.\d+$/;
+if (!releaseVersionRegex.test(releaseVersion)) {
+  console.log(
+    "error: invalid -releaseVersion; must match regular expression: " +
+      releaseVersionRegex
+  );
+  return 0;
+}
+
+const releasesDateRegex = /^\d{4}-\d{2}-\d{2}$/;
+if (!releasesDateRegex.test(releaseDate)) {
+  console.log(
+    "error: invalid -releaseDate; must match regular expression: " +
+      releasesDateRegex
+  );
+  return 0;
+}
+
+let data = fs.readFileSync("quick-lint-js-template.json", "utf8");
+data = data.replace(/{version}/g, releaseVersion);
+data = data.replace(/\${releaseDate}/g, releaseDate);
```

Comment:
Why do we use `{}` in one case and `${}` in another case? We should be consistent.